generators/client/templates/react/src/main/webapp/app/modules/account/settings/settings.reducer.ts.ejs

```diff
@@ -86,6 +86,9 @@ export const saveAccountSettings = account => async <% if (enableTranslation) {
     }
     <% } %>  
   await dispatch(getSession());
+  <% if (enableTranslation) { %>
+    document.querySelector('html').setAttribute('dir', isRTL(account.langKey) ? 'rtl' : 'ltr');
```

Comment:
@Hawkurane :
this condition will fail for some config, as isRTL won't be available
maybe use the same condition as above `enableTranslation && enableI18nTL` ?